bigquery/src/clojure/gclouj/bigquery.clj

```diff
@@ -252,6 +252,8 @@
       (.dryRun builder dry-run?))
     (when max-wait-millis
       (.maxWaitTime builder max-wait-millis))
+    (when-not (nil? use-legacy-sql?)
```

Comment:
As is, passing `use-legacy-sql? = false` would make it use legacy sql, no?

Reply:
Yeah, as per @emilssolmanis, given the truthiness (well, I guess falseness ;) of nil it would need to be this instead?

```suggestion
    (when-not use-legacy-sql?
```

Reply:
Is standard SQL supported in `com.google.cloud/gcloud-java-bigquery "0.2.8"`, I'd expect we should update those libs at some point too!

Reply:
You're both correct (suggestion applied).

Standard is supported (it's already an option for query-jobs, just not queries).

I did look at trying an upgrade to the library version but given we're currently ~0.65 versions behind I didn't want to give it a go right now.

Reply:
Ha! Yeah, doesn't need to be in this PR. I've created #5 to track in case a benevolent soul comes along to help.